bluebottle/suggestions/views.py

```diff
@@ -29,13 +29,13 @@ def get_queryset(self):
 
 class SuggestionDetail(generics.RetrieveUpdateAPIView):
     queryset = Suggestion.objects.all()
-    permission_classes = (IsAuthenticated,)
+    permission_classes = (AllowAny,)
```

Comment:
Is this a potential security issue? Anybody can read update and delete suggestions. That does not seem right.